packages/grafana-ui/src/components/PageLayout/PageToolbar.tsx

```diff
@@ -80,18 +80,21 @@ export const PageToolbar: FC<Props> = React.memo(
               )}
             </>
           )}
-          {titleHref && (
-            <h1 className={styles.h1Styles}>
-              <Link
-                aria-label="Search dashboard by name"
-                className={cx(styles.titleText, styles.titleLink)}
-                href={titleHref}
-              >
-                {title}
-              </Link>
-            </h1>
-          )}
-          {!titleHref && <h1 className={styles.titleText}>{title}</h1>}
+          {title ? (
+            titleHref ? (
+              <h1 className={styles.h1Styles}>
+                <Link
+                  aria-label="Search dashboard by name"
+                  className={cx(styles.titleText, styles.titleLink)}
+                  href={titleHref}
+                >
+                  {title}
+                </Link>
+              </h1>
+            ) : (
+              <h1 className={styles.titleText}>{title}</h1>
+            )
+          ) : null}
```

Comment:
wonder if there is way to simplify this using a function or something, I find one `? :` expression in a render path hard to read, two nested even more so :)

Reply:
the outer ? : null seems unnecessary as well (just title && ...) should do

Reply:
yeah, it's pretty hard to read, on it

Reply:
simpler approach in 24823eed9cd387843fb2b972e3c56164512ab826. i feel there's a bit of room for improvement here, also because this component is very specific to dashboards ( title's link explicitly mention dashboards, as well as parent's link)

I think we might benefit from a more generic breadcrumbs component here, I've seen some mockups in some figma explorations. do you think we can tackle this in a follow-up pr? do you know if we are already planning to introduce such component?

Reply:
> the outer ? : null seems unnecessary as well (just title && ...) should do

I read this too late :D I went for avoiding the ternary operator altogether as it felt cleaner instead of
```tsx
{title && (titleHref ? <>a link</> : <>not a link</>} }
```

but happy to change if you think it's better